enhanced/topbar.py

```
@@ -52,10 +52,10 @@ def get_welcome_image():
         file_welcome = welcomes[0]
 #        file_welcome = os.path.join(path_welcome, 'welcome.jpg')
     return file_welcome
-
+#added change
 def get_preset_name_list():
     path_preset = os.path.abspath(f'./presets/')
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)